packages/app-store/routing-forms/pages/forms/[...appPages].tsx

```diff
@@ -109,7 +111,11 @@ export default function RoutingForms({
       description: t("download_responses_description"),
     },
   ];
-
+ const validateFormName = (name: string): void => {
+    if (isFormNameEmpty(name)) {
+      throw new Error("Form name is required");
+    }
+  };
```

Comment:
Hey @Aditay2002 
Is this fix for issue #9871?
Could you please share screenshots or some video for this?
I was also looking into the same issue and I am not sure how this fixes the problem.